xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererUtil.cpp

```diff
@@ -24,7 +24,7 @@
 #include "cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayImage.h"
 #include "cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySpu.h"
 #include "cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h"
-#include "windowing/WindowingFactory.h"
+#include "windowing/WinSystem.h"
```

Comment:
No other changes so have to ask; is this include necessary?

Reply:
probably not, thanks